compose_generator/odcs_fetcher.py

```diff
@@ -17,5 +21,21 @@ class ODCSFetcher(ComposeFetcher):
     """
     Fetch ODCS compose based on a remote compose-reference and store it locally
     """
-    def __call__(self, request_reference: ComposeReference) -> ODCSResultReference:
-        raise NotImplementedError()
+
+    def __call__(
+        self, request_reference: ComposeReference, download_path: Optional[Path] = None
+    ) -> ODCSResultReference:
+        if download_path is not None:
+            download_path = tempfile.mkdtemp()
+        download_path.parent.mkdir(parents=True, exist_ok=True)
+
+        compose_path = download_path / "ocds_compose"
+        response = requests.get(request_reference.url)
+        if response.status_code == 200:
```

Comment:
If it's not, then we return a path that contains nothing. We should fail somehow instead.

Reply:
right, added an assertion